### DISCUSSION TITLE
Structure file of Tree Simulation (Pine) And how to start simulation with my own .stl file

### DISCUSSION THREAD

**Discussion by wuwuyiyirong5** [InteractiveComputerGraphics/PositionBasedDynamics#119] [Q&A]
The structure file of Tree Simulation (Pine) can be shared with me? How to start simulation with my own .stl file? Thanks a lot!

**Comment by janbender** [comment]
In fact I don't know where or if we still have the pine model someehere. However, these tree models can easily be generated with a Blender plugin. Maybe you can try this. 

If you want to use your STL model, you can either convert it to an OBJ and load it in a scene file or implement an STL importer.
